content/en/project/BLUP_Brain-Learning-Unicorn-Project/index.md

```diff
@@ -0,0 +1,233 @@
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects..
+
+summary: "Can a model predict the genetic profile of an individual based on brain regions volumes? There is growing evidence suggesting that genetic variations formally associated to neurodevelopmental disorders have significant effects on brain structures. In this project, the performance of three classifiers will be compared when predicting the genetic status of individuals from brain region volumes in a highly imbalanced dataset (UK BioBank cohort)."
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: "BLUPproject_website.png"
+---
+<!-- This is an html comment and this won't appear in the rendered page. You are now editing the "content" area, the core of your description. Everything that you can do in markdown is allowed below. We added a couple of comments to guide your through documenting your progress. -->
+
+# BLUP: Brain Learning Unicorn Project
+
+Click [HERE](https://brainhack-school2020.github.io/EliseD_BLUP_Brain-Learning-Unicorn-Project/) to see this readme as a website.
+
+### ***Can a model predict the genetic profile of an individual based on brain regions volumes?***
+
+*Trying to deal with new stuff learned at the amazing Brainhack school*
+
+<ins>Author:</ins> Elise Alix Douard
+
+<ins>Other project contributors:</ins>
+BHS, Hannah Kiesow [@hannahmaykiesow](https://twitter.com/hannahmaykiesow) (also working on UKBiobank), Kuldeep Kumar [@meetkd007](https://twitter.com/meetkd007) (extracting data from UKBB servers)
+
+## Personal presentation
+
+Welcome to this project dear unicorn student ! 
+
+<p align="center">
+<img src="https://media.giphy.com/media/CzQ9Kl1UIt8hG/giphy.gif">
+</p>
+<p> <font size="1.5"> Source: https://media.giphy.com/</font></p> 
+
+I am Elise, Ph.D. student in neurosciences at the UdeM since near to 4 years, and working on **the contribution of genetic to neurodevelopmental disorders** as autism. I don't really fit in a specific domain (genetic/cognitive neurosciences/psychology). I guess it is what we call a *unicorn student*? Currently, I am working with genetic data (Copy Number Variants), clinical phenotypes and doing a lot of statistics and graphs on R. But my initial formation was in cognitive neurosciences where I started to work with multimodal data (combination of Arterial Spin Labelling MRI data and Eye-tracking data). 
+
+Since I started my Ph.D., I never used MRI data nor python, and I am here to take a revenge on that. 
+
+**Skills:**
+
+- Data management (feed me with multimodal data plz)
+- Statistics
+- Debugging codes
+
+# Project definition
+
+## Background
+
+<p align="center">
+<img src="Slide_content/illustration_genetic.png">
```

Comment:
This image is missing. It should be added in the project repo and can be referenced as `<img src="illustration_genetic.png">`. As long as the image is here, I'll be able to fix the link even if it's broken anyway.